src/test/resources/json/module.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
You need to adopt your json field names according to the endpoint specification based on the software architecture document.